composer.json

```diff
@@ -46,6 +47,7 @@
         "preferred-install": "dist"
     },
     "extra": {
+        "contao-manager-plugin": "Contao\\FaqBundle\\ContaoManager\\Plugin",
```

Comment:
why would that make sense? it would be error-prone for the user, and checking it is a matter of array-search. That information is already available when searching for packages.